package test.oob.exception;

public class ExceptionThrowDemo {

	
	
	public static void fangfa1(){
		
		
		
		System.out.println();
	}
	
	
	
	
	public static void main(String[] args) {

		
		
		
	}

}
